instrumentation/opentelemetry-instrumentation-aio-pika/pyproject.toml

```diff
@@ -29,7 +29,7 @@ dependencies = [
   "wrapt >= 1.0.0, < 2.0.0",
 ]
 
-[project.optional-dependencies]
+[tool.opentelemetry-bootstrap]
```

Comment:
Maybe this is breaking `instrumentation.dependencies.get_dist_dependency_conflicts`

Reply:
Yeah, it' breaking it. Given that AFAICS [tool.*] metadata is not available in installed package I guess we can use the `_package` module inside each instrumentation and maybe make `[tool.opentelemetry-bootstrap]` dynamic to avoid duplicating the data.

`instrumentation.dependencies.get_dist_dependency_conflicts` would then be updated to look at the _package module instead of the distribution requirements.